stackdriver_exporter.go

```diff
@@ -87,9 +87,9 @@ var (
 
 	// Monitoring collector flags
 
-	monitoringMetricsTypePrefixes = kingpin.Flag(
-		"monitoring.metrics-type-prefixes", "Comma separated Google Stackdriver Monitoring Metric Type prefixes.",
-	).Required().String()
+	monitoringMetricsPrefixes = kingpin.Flag(
```

Comment:
Thinking about this some more, I think we should add this as a new flag and keep the old flag for at least one release.

We can still combine the data so both flags work. But if the old flag is used, we print a warning in the log.